applets/notification_area/status-notifier/sn-item-v0.c

```diff
@@ -329,6 +330,7 @@ update (SnItemV0 *v0)
       gtk_widget_set_tooltip_markup (GTK_WIDGET (v0), NULL);
     }
 
+ gtk_button_set_label (GTK_BUTTON (v0), v0->label);
```

Comment:
I don't think it should be called unconditionally.

Reply:
I think it's unconditional